From f68c561aafc704d46ccf780e32812ffc81d4be6c Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Fri, 9 Mar 2018 11:04:18 +0800
Subject: [PATCH 3467/5242] MLK-17736-02 dts: imx7ulp-evk: add interrupt
 property for rpmsg io node

commit  0bcad3519347cef4f6393f2cfb071c8435f11347 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add interrupt property for rpmsg io node.

Reviewed-by: Robin Gong<yibin.gong@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 18b7301..37e28ea 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -158,6 +158,9 @@
 		port_idx = <0>;
 		gpio-controller;
 		#gpio-cells = <2>;
+		#interrupt-cells = <2>;
+		interrupt-controller;
+		interrupt-parent = <&rpmsg_gpio0>;
 		status = "okay";
 	};
 
@@ -166,6 +169,9 @@
 		port_idx = <1>;
 		gpio-controller;
 		#gpio-cells = <2>;
+		#interrupt-cells = <2>;
+		interrupt-controller;
+		interrupt-parent = <&rpmsg_gpio1>;
 		status = "okay";
 	};
 
-- 
1.7.9.5

